lib/core/rancher/step.sh

```diff
@@ -94,21 +95,27 @@ function ih::setup::core.rancher::install() {
         fi
     done
 
-    rm -rf ~/.rd
-    $(/Applications/Rancher\ Desktop.app/Contents/Resources/resources/darwin/bin/rdctl start)
-    # Check for docker binary in /usr/local/bin
-    # Check if /usr/local/bin/docker exists
-    DOCKERBIN=/usr/local/bin/docker
-    DOCKERCOMPOSEBIN=/usr/local/bin/docker-compose
-    if [ ! -f "$DOCKERBIN" ]; then
-
-        echo "In order to continue with Rancher configuration and be able to use this engine, some IDEs require the creation of symlinks for remote Python interpreters"
-        echo "Your password is required for the creation of symlink mentioned above"
-        sudo ln -s $HOME/.rd/bin/docker /usr/local/bin/docker
-        if [ ! -f "$DOCKERCOMPOSEBIN" ]; then
-            sudo ln -s $HOME/.rd/bin/docker-compose /usr/local/bin/docker-compose
+    # Continue with the setting just if the cask was installed successfully
+    if [ $CASKSUCCEEDED -eq 0 ]; then
```

Comment:
Maybe we should error out here
```shell
if [ $CASKSUCCEEDED -eq 0 ]; then
  ih::log::warn "Could not install Rancher Desktop"
  return 1
fi
```

Reply:
Arg, this comment is in the wrong spot, and is wrong. It should have been on line 99, and been like this:

```shell
if [ $CASKSUCCEEDED -ne 0 ]; then
  ih::log::warn "Could not install Rancher Desktop"
  return 1
fi
```